mockall/tests/mock_generic_struct_with_generic_static_method.rs

```diff
@@ -6,12 +6,12 @@ use mockall::*;
 use std::sync::Mutex;
 
 mock! {
-    Foo<T: 'static> {
-        fn foo<Q: 'static>(t: T, q: Q) -> u64;
+    Foo<T: 'static + std::fmt::Debug> {
```

Comment:
Why do you need to add `Debug`?  Shouldn't the autoderef magic pretty-print the panic message even without this trait bound?

Reply:
unfortunately not, removing the bounds falls back to the `"?"` message. not sure if i'm missing anything else to make it work